src/public_parameters/registry.rs

```diff
@@ -50,13 +51,13 @@ impl Registry {
         let key = format!("public-params-rc-{rc}-coproc-{lang_key}");
         // read the file if it exists, otherwise initialize
         if let Some(pp) = disk_cache.get::<PublicParams<'static, C>>(&key) {
-            eprintln!("Using disk-cached public params for lang {}", lang_key);
+            info!("Using disk-cached public params for lang {}", lang_key);
             Ok(Arc::new(pp))
         } else {
             let pp = default(lang);
             disk_cache
                 .set(key, &*pp)
-                .tap_ok(|_| eprintln!("Writing public params to disk-cache: {}", lang_key))
+                .tap_ok(|_| info!("Writing public params to disk-cache: {}", lang_key))
```

Comment:
This is good, but we may want to look for instances of `dbg!` as well and replace them with error logs.